.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+<!-- If this PR closes a JIRA ticket, make sure the title starts with the JIRA issue number,
+for example HLA-1234: <Fix a bug> -->
+Resolves [HLA-nnnn](https://jira.stsci.edu/browse/HLA-nnnn)
+
+<!-- If this PR closes a GitHub issue, reference it here by its number -->
+Closes #
+
+<!-- describe the changes comprising this PR here -->
+This PR addresses ...
+
+**Checklist for maintainers**
+- [ ] added entry in `CHANGES.rst` within the relevant release section
```

Comment:
Minor point, but in drizzlepac this is CHANGELOG.rst.

Reply:
done